meson.build

```diff
@@ -300,15 +300,15 @@ if not tree_sitter_dep.found()
   meson.override_dependency('tree-sitter', tree_sitter_dep)
 endif
 
-# handle tree-sitter-c
-r = run_command(py3_exe, check_meson_subproject_py, 'tree-sitter-c', check: false)
+# handle rizin-grammar-c
+r = run_command(py3_exe, check_meson_subproject_py, 'rizin-grammar-c', check: false)
 if r.returncode() == 1 and get_option('subprojects_check')
   error(subproject_clean_error_msg)
 endif
 
-tree_sitter_c_dep = dependency('tree-sitter-c', required: get_option('use_sys_tree_sitter'), static: is_static_build, fallback: [])
+tree_sitter_c_dep = dependency('rizin-grammar-c', required: get_option('use_sys_tree_sitter'), static: is_static_build, fallback: [])
```

Comment:
just rename it as `rz-grammar` instead of `rizin-grammar-c`

Reply:
Too ambiguous, IMHO.

Reply:
ok, but also the binary called has been renamed.